index.html

```diff
@@ -22,6 +22,20 @@
     <link type="text/css" rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
     <link type="text/css" rel="stylesheet" href="styles/main.css">
 
+    <!-- Leaflet map, js plugins, and css & sources -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.1/dist/leaflet.css" />
+    <script src="https://unpkg.com/leaflet@1.0.1/dist/leaflet.js"></script>
+    <script src="https://api.mapbox.com/mapbox.js/plugins/leaflet-omnivore/v0.2.0/leaflet-omnivore.min.js"></script>
+    <!-- Search tool for map -->
+    <script src="carpool-map/js/l.control.geosearch.js"></script>
+    <script src="carpool-map/js/l.geosearch.provider.openstreetmap.js"></script>
+    <link rel="stylesheet" href="carpool-map/css/l.geosearch.css"/>
+    <!-- JSON parser to GeoJSON by Casey Thomas -->
+    <script src="carpool-map/js/geojson.js"></script>  
```

Comment:
Do you reckon we could move all these scripts down to the bottom of the page, immediately above `<script src="carpool-map/js/map.js"></script>`? 

Basically, there's quite a lot of assets here and I want to prevent them from blocking other assets or affecting the initial page load/render speed. They're only needed for the map, which is quite a ways down the page, so new users will take a least a few extra seconds to scroll down there, which gives us a bit more breathing room to load these assets. Hence, while it's important that they be loaded, they're a lower priority than some of the other scripts on the page, so I recommend that they be loaded last. Does that sound okay to you?